applications/mne_scan/plugins/ftbuffer/ftconnector.cpp

```diff
@@ -612,3 +601,43 @@ FIFFLIB::FiffInfo FtConnector::infoFromSimpleHeader()
 
     return defaultInfo;
 }
+
+//=============================================================================================================
+
+void FtConnector::checkForMissingMetadataFields(MetaData& data) const
+{
+    if (!data.bFiffInfo){
+        data.setFiffinfo(infoFromSimpleHeader());
+    } else {
+        if ( data.info.meas_date[0] == -1 )
+        {
+            data.info.meas_date[0] = static_cast<int32_t>(QDateTime::currentDateTime().toSecsSinceEpoch());
+            data.info.meas_date[1] = 0;
+        }
+
+        if ( data.info.sfreq <= 0 )
```

Comment:
as long as we have not set on a specific formatting rule, I would propose follow the old style aka no spaces before and after `(`as well as `)`